index.html

```diff
@@ -138,7 +138,10 @@ <h5 class="font-open-sans sm-text-center">Blockcast provides a fast, uncensored
                                       <input type="submit" value="Join The Waitlist" name="subscribe" id="mc-embedded-subscribe" class="button btn btn-danger btn-cons btn-signup">
                                     </span>
                                   </div>
-                                  <p class="fs-12 m-r-20">Be first to find out when we Launch our product.</p>
+                                  <p class="fs-12 m-r-20">
+                                    Share your location to learn about service in your region <input type="checkbox" name=location" /><br />
+                                    Be the first to find out when we launch our product.
+                                  </p>
```

Comment:
I already add countries into the signup form

Reply:
Ok merge and replace with your better one